flutter_application_1/lib/bluetooth_manager.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter_bluetooth_basic/flutter_bluetooth_basic.dart';
+
+class BluetoothDeviceManager {
+  final BluetoothManager _bluetoothManager = BluetoothManager.instance;
+
+  void startScan({required Duration timeout}) {
+    _bluetoothManager.startScan(timeout: timeout);
+  }
+
+  Stream<List<BluetoothDevice>> get scanResults =>
+      _bluetoothManager.scanResults;
+
+  Future<void> connect(BluetoothDevice device) async {
+    await _bluetoothManager.connect(device);
+  }
```

Comment:
May try `broadcast` to see other devices. Two devices advertising may not be able to see each other